src/bubo.cc

```diff
@@ -55,8 +55,19 @@ NAN_METHOD(Bubo::Initialize)
         return;
     }
 
-    Local<Value> ignoredVal = Nan::Get(opts, ignoredAttributes).ToLocalChecked();
-    Local<Object> ignored = Nan::To<Object>(ignoredVal).ToLocalChecked();
+    Local<Value> ignored_value = Nan::Get(opts, ignoredAttributes).ToLocalChecked();
+    Local<Object> ignored_array = Nan::To<Object>(ignored_value).ToLocalChecked();
```

Comment:
Actually it's because v8::Value doesn't actually expose a ToArray function for some reason: http://izs.me/v8-docs/classv8_1_1Value.html.

Odd because there is an IsArray function...